openpype/hosts/hiero/api/plugin.py

```diff
@@ -388,6 +388,7 @@ def __init__(self, cls, context, **options):
         self.__dict__.update(cls.__dict__)
         self.context = context
         self.active_project = lib.get_current_project()
+        self.fname = path
```

Comment:
I can't seem to find where this is being used?

Reply:
Ok, this is for backwards compatibility from what I can gather.

Reply:
Actually - this isn't a Loader plug-in but some `ClipLoader` class that some host integrations seemed to have implemented. This was implemented so that it'd explicitly set the path for the `ClipLoader` instead of trying to search for `loader.fname`.